project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -137,4 +137,15 @@ PlaceDetails getPlaceDetails(PlaceDetailsRequest request)
             throws ApiException, InterruptedException, IOException {
         return request.await();
     }
+
+    private String genCuisinesStr(List<String> cuisineLst) {
+        String cuisineStr = "";
```

Comment:
The whole method can be simplified by staticlly importing:
import static java.util.stream.Collectors.joining;

and then:
return CUISINES.stream().collect(joining("|"));

Always try to search for a utility to do simple things for you:)

Reply:
Thanks, I don't naturally think about using streams and other java utilities, I will keep it in mind fot next times.